data/json/npcs/BGSS_talk.json

```diff
@@ -0,0 +1,922 @@
+    ]
+},
+{
+    "id": "BGSS_RELIGIOUS_2_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "Same as anyone. I turned away from God, and now I'm paying the price. The Rapture has come, and I was left behind. So now, I guess I wander through Hell on Earth. I wish I'd paid more attention in Sunday School.",
+    "responses": [
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+},
+{
+    "id": "BGSS_EVACUEE_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "I made it to one of those evac shelters, but it was almost worse than what I left behind. Escaped from there, been on the run since.",
+    "responses": [
+      { "text": "How did you survive on the run?", "topic": "BGSS_EVACUEE_1_STORY2" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+},
+{
+    "id": "BGSS_EVACUEE_1_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "I spent a lot of time rummaging for rhubarb and bits of vegetables in the forest before I found the courage to start picking off some of those dead monsters. I guess I was getting desperate.",
+    "responses": [
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+},
+{
+    "id": "BGSS_EVACUEE_2_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "Same as most people who didn't get killed straight up during the riots. I went to one of those stupid fucking evacuation death traps. I actually lived there for a while with three others. One guy who I guess had watched a lot of movies kinda ran the show, because he seemed to really know what was going on. Spoiler alert: he fucking didn't.",
+    "responses": [
+      { "text": "What happened to your original crew?", 
+        "trial": { "type": "PERSUADE", "difficulty": 30, "mod": [ [ "BRAVERY", 1 ], [ "TRUST", 1 ] ] },
+        "success": { "topic": "BGSS_EVACUEE_2_STORY2" },
+        "failure": { "topic": "BGSS_EVACUEE_2_NOTYET" }
+      },
+      { "text": "Never mind...", "topic": "TALK_FRIEND" }
+    ]
+},
+{
+    "id": "BGSS_EVACUEE_2_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "Things went south when our fearless leader decided we had to put down one of the other survivors that had been bitten. Her husband felt a bit strongly against that, and I wasn't too keen on it either; by this point, he'd already been wrong about a lot. Well, he took matters into his own hands and killed her. Then her husband decided one good turn deserves anotherb, and killed the idiot. And then she got back up and I killed her again, and pulped our former leader. Unfortunately she'd given her husband a hell of a nip during the struggle, when he couldn't get his shit together enough to fight back. Not that I fucking blame him. We made it out of there together, but it was too much for him, he clearly wasn't in it anymore... The bite got infected, but it was another zombie that finally killed him. And then I was alone.",
```

Comment:
Replace "good turn deserves anotherb" with "good turn deserves another".

Reply:
I felt that the extra "b" really gave it flare